app/Resources/views/article-text.html.twig

```diff
@@ -18,6 +18,12 @@
 
         {% block main %}
 
+            {% if viewSelector %}
```

Comment:
Technically the behaviour is required for having collapsible article sections, not the view selector? (The view selector isn't actually optional on article text pages too.)